app/components/works/title_component.html.erb

```diff
@@ -19,6 +19,5 @@
     </div>
 
     <%= render Works::ContactEmailComponent.new(form:, key: 'work.contact_email') %>
-    </legend>
```

Comment:
Assume also this is a11y-related? Or maybe the opening `<legend>` element isn't there?